src/app/src/pages/eval/components/Eval.tsx

```diff
@@ -54,8 +60,13 @@ export default function Eval({
   const fetchEvalById = React.useCallback(
     async (id: string) => {
       const resp = await callApi(`/results/${id}`, { cache: 'no-store' });
+      if (!resp.ok) {
```

Comment:
out of curiosity, when does this happen? Should we put improving this flow on the backlog?

Reply:
The only case I ran into is when it doesn't exist